Runtime/Triangulator.cs

```diff
@@ -1790,6 +1808,8 @@ public void Execute(Allocator allocator, bool refineMesh, bool constrainBoundary
                     return;
                 }
 
+                if (typeof(T) == typeof(int)) throw new NotSupportedException("Mesh refinement is not supported for integer types.");
```

Comment:
I added a different test for verifying the error message.
I still feel like ignoring these tests is appropriate, because the tests don't actually run.